bot/cogs/last_fm.py

```diff
@@ -0,0 +1,206 @@
+                try:
+                    embed.add_field(name='Release Data', value=data['album']['wiki']['published'], inline=False)
+                except KeyError:
+                    pass
+                try:
+                    embed.add_field(name='About', value=data['album']['wiki']['summary'].split('<a', 1)[0])
+                except KeyError:
+                    pass
+                await ctx.send(embed=embed)
+        except IndexError:
+                await ctx.send('Invalid search term, try again')
+
+    @commands.command(
+      name="getartist",
+      brief="A command that get's info about an artist",
+      description="Takes in just artist name",
+      aliases=['artist', 'getar']
+    )
+    async def getartist(self, ctx, *, args):
+        try:
+                data = get_artist(args)
+                album_data = get_data('artist.gettopalbums&artist='+urllib.parse.quote(data['name']))['topalbums']['album']
+                bio = data['bio']['summary'].split('\n', 2)[0].split('<a', 1)[0]
+                if bio == '':
+                    bio = data['bio']['summary'].split('\n', 2)[1].split('<a', 1)[0]
+                embed = discord.Embed(title=data['name'], url=data['url'], description=bio)
+                embed.set_thumbnail(url=album_data[0]['image'][2]['#text'])
+                top_albums = "\n".join([x["name"] for x in album_data[:10]])
+                similar = "\n".join([x["name"] for x in data["similar"]["artist"]])
+                if not similar:
+                    similar = 'No Similar Artists Avaliable'
+                embed.add_field(name='Top Albums', value=top_albums, inline=True)
+                embed.add_field(name='Similar Artists', value=similar, inline=True)
+                await ctx.send(embed=embed)
+        except IndexError:
+                await ctx.send('Invalid search term, try again')
+
+    @commands.command(
+      name="topsongs",
+      brief="Gets a list of the top songs on the world charts",
+      aliases=['songs', 'tops']
+    )
+    async def topsongs(self, ctx):
+        data = top_tracks()
+        embed = discord.Embed(title='Top 10 Tracks', url='https://www.last.fm/charts')
+        for count, song in enumerate(data[:10], 1):
+                embed.add_field(name=count, value=f"{song['name']} by {song['artist']['name']}", inline=False)
+        await ctx.send(embed=embed)
+
+    @commands.command(
+      name="topartists",
+      brief="Gets a list of the top artists on the world charts",
+      aliases=['artists', 'topa']
+    )
+    async def topartists(self, ctx):
+        data = top_artists()
+        embed = discord.Embed(title='Top 10 Artists', url='https://www.last.fm/charts')
+        for count, artist in enumerate(data[:10], 1):
+                embed.add_field(name=count, value=artist['name'], inline=False)
+        await ctx.send(embed=embed)
+
+
+def setup(bot):
+    "Imports the cog"
+    bot.add_cog(MusicCog(bot))
+
+
+def get_data(method, method2=''):
```

Comment:
Could benefit from type annotation, i.e `method: str, method2: str`. I am also not a huge fan of the naming here, `method2` is a little confusing as to what it does just from reading (I am just being nit-picky here to try to get the code perfect, haha).

Pretty good docstring on this one.